### PR TITLE
Fix certs readme

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

File: sdk/keyvault/azure-keyvault-certificates/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ and other secrets
 ## Getting started
 ### Install the package
 Install [azure-keyvault-certificates][pypi_package_certificates] and
-[azure-identity][azure_identity] with [pip][pip]:
+[azure-identity][azure_identity_pip] with [pip][pip]:
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/README.md]: Nitpick: PyPI makes the package available, and pip isn't required to acquire it. So if you want a new name for this link I think `azure_identity_pypi` is more to the point (though I'd just leave it as it was). Anyway that's a nitpick 😊

File: sdk/keyvault/azure-keyvault-keys/README.md [hunk 2]
```diff
@@ -403,17 +404,22 @@ see the Code of Conduct FAQ or contact opencode@microsoft.com with any
...
-[JWK]: https://tools.ietf.org/html/rfc7517
-[keyvault_docs]: https://docs.microsoft.com/en-us/azure/key-vault/
 [backup_operations_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/samples/backup_restore_operations.py
 [backup_operations_async_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/samples/backup_restore_operations_async.py
+[list_operations_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/samples/list_operations.py
+[list_operations_async_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/samples/list_operations_async.py
+[recover_purge_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/samples/recover_purge_operations.py
+[recover_purge_async_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/samples/recover_purge_operations_async.py
+[JWK]: https://tools.ietf.org/html/rfc7517
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-keys/README.md]: unused

File: sdk/keyvault/azure-keyvault-certificates/samples/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+* [hello_world.py][hello_world_sample] and [hello_world_async.py][hello_world_async_sample] - create/get/update/delete certificates
+* [backup_restore_operations.py][backup_operations_sample] and [backup_restore_operations_async.py][backup_operations_async_sample] - backup and
+recover certificates
+* [list_operations.py][list_operations_sample] and [list_operations_async.py][list_operations_async_sample] - list certificates
+* [recover_purge_operations.py][recover_purge_operations_sample] and [recover_purge_operations_async.py][recover_purge_operations_async_sample] - recover and purge certificates
+* [issuers.py][issuers_sample] and [issuers_async.py][issuers_async_sample] - manage certificate issuers
+* [contacts.py][contacts_sample] and [contacts_async.py][contacts_async_sample] - manage certificate contacts
+
+[backup_operations_sample]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations.py
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/samples/README.md]: GitHub redirects from /tree/master to /blob/master

File: sdk/keyvault/azure-keyvault-certificates/README.md (hunk 1 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/README.md]: thanks, i'll go ahead and change it bc im doing other changes too

**Pull request merged**